build.gradle.kts

```diff
@@ -47,8 +48,15 @@ tasks.getByName<org.jetbrains.intellij.tasks.RunPluginVerifierTask>("runPluginVe
         "2023.1.4",
         "2023.1.5",
         "2023.2",
+        "2023.2.1",
+        "2023.2.2",
+        "2023.2.3",
+        "2023.2.4",
+        "2023.2.5",
+        "2023.3",
+        "2023.3.1",
```

Comment:
Might as well add `2023.3.2`, as it's listed [here](https://data.services.jetbrains.com/products?code=DG%2CPS%2CDL%2CMPS%2CMPSIIP%2CRDCPPP%2CWS%2CAC%2CIIC%2CIIE%2CGO%2CCL%2CPCD%2CPCC%2CRD%2CPCE%2CIIU%2CRM%2CPCP&fields=code%2Cname%2Creleases.downloads%2Creleases.version%2Creleases.build%2Creleases.type)

Expand node `15`
Expand `releases` sub-node, and `0` below that
...and you'll see `2023.3.2` as a listed entry

Reply:
It's early to add 2023.3.2, because is Release Candidate now

Will add this version after IDEA release

Reply:
Ok